ProjFS.Mac/PrjFSKextTests/VnodeCacheTests.mm

```diff
@@ -594,6 +598,7 @@ - (void)testFindVnodeRootFromDiskAndUpdateCache_InvalidUpdateCacheBehaviorValue
     // In production FindVnodeRootFromDiskAndUpdateCache would panic when it sees an
     // invalid UpdateCacheBehavior, but in user-mode the assertf if a no-op
     VirtualizationRootHandle rootHandle;
+    [self setExpectedFailedKextAssertionCount:1];
```

Comment:
So this means we will hit exactly one assert in the called test code below?

To confirm asserts in test cases don't cause failures?

Reply:
Failed assertions in kext code when run in a test *will* now cause test failures, unless explicitly declared as expected as above.

If in the course of running this particular test case, 0 assertions failed in kext code, the test will fail. Likewise if 2 assertions fail.

Reply:
No, failed `assert()`s previously and now cause a process termination. (via `abort()` I think) Not ideal, and something I'll fix up at some point - probably by defining our own assertion macro that can be used outside the test class. (You can't use the `XCTAssert*` macros outside the test itself because their expansion references `self`.)